src/demo/demoutils.ts

```diff
@@ -1,32 +1,17 @@
-import * as path from 'path'
-import * as fs from 'fs'
-const Web3 = require('web3')
+import { ethers } from 'ethers'
+import { PreciseProofCommitmentRegistryFactory } from '../ethers/PreciseProofCommitmentRegistryFactory'
+import { BigNumber, bigNumberify } from 'ethers/utils'
 
 interface Transaction {
     from: string
 } 
 
-export const demoinput = {
```

Comment:
what happened to this guy?

Reply:
Our use case changed so I changed this too. Can return some of these properties if needed